generators/server/templates/src/main/java/package/domain/_Authority.java

```diff
@@ -29,6 +29,13 @@
     @Column(length = 50)<% } %>
     private String name;
 
+    public Authority() {
```

Comment:
I had kept the empty constructor because Jackson needs it (or you need to specify the constructor explicitly). I don't know if Jackson is used here so maybe that was useless.